libexec/rtld-elf/aarch64/reloc.c

```diff
@@ -65,6 +71,109 @@ init_pltgot(Obj_Entry *obj)
 	}
 }
 
+#if __has_feature(capabilities)
+/*
+ * Fragments consist of a 64-bit address followed by a 56-bit length and an
+ * 8-bit permission field.
+ */
```

Comment:
Strange they went with this rather than splatting out the capability bits with a zero base and letting it be adjusted at run time... this kind of thing comes from the __caprelocs world from before CBuildCap/BUILD was even dreamt up.